model/db.js

```diff
@@ -166,6 +167,16 @@ var SupportingFilesSchema = new Schema({
   su_date: {type: Date, required: true, default: Date.now()}
 });
 
+var CalculationSchema = new Schema({
+  in_inventory: {type: Schema.Types.ObjectId, required: true, ref: 'Inventory'},
+  se_sector: {type: Schema.Types.ObjectId, required: true, ref: 'Sector'},
+  emission_factor: {type: Schema.Types.ObjectId, ref: 'Data'},
+  ac_activity: {type: Schema.Types.ObjectId, ref: 'Data'},
+  un_unit: {type: Schema.Types.ObjectId, required: false, ref: 'Unit'},
+  calculation_value: {type: Number, required: true},
+  da_data_modified: {type: Date, default: Date.now()}
```

Comment:
Hey Andre... could you also add the uncertainty values? min and max?  It would also be calculated and stored.